packages/eds-lab-react/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -265,7 +270,7 @@ function ComboboxInner<T>(
     setSelectedItems,
   } = useMultipleSelection(multipleSelectionProps)
 
-  let comboBoxProps: UseComboboxProps<ComboboxOption<T>> = {
+  let comboBoxProps: UseComboboxProps<AutocompleteOption<T>> = {
```

Comment:
Forgot this?

Reply:
These are the props/options that goes to downshifts `useCombobox` hook.

Reply:
Yea I just realized looking a bit more closely